controllers/homeRoutes.js

```diff
@@ -78,45 +78,39 @@ router.get('/feed', withAuth, async (req, res) => {
 });
 
 // github repo search page
-router.get('/search', withAuth, async (req, res) => {
-  const query = req.query.query;
-
-  if(query){
-    try {
-      const response = await octokit.request("GET /search/repositories", {
-        q: query,
-        page: 1, // Replace with the desired page number
-        per_page: 10, // Replace with the desired number of results per page
-        sort: "stars", // Replace with your preferred sorting criteria
-        order: "desc" // Replace with "asc" for ascending order or "desc" for descending order
-      });
+router.get('/reposearch', withAuth, async (req, res) => {
   
-      // Extract the list of repositories from the response
-      const repositories = response.data.items;
-      res.render('search', {
-        repositories,
-        title: 'Search Repos',
-        style: 'dashboard.css',
-        logged_in: req.session.logged_in
-      });
-    } catch (error) {
-      console.error(error);
-    }
-  } else {
-    res.render('search', {
-      title: 'Search Repos',
-      logged_in: req.session.logged_in
-    });
+  let repositories = [];
+  repositories = req.session.repos;
+
+  const renderData = {
+    title: 'Search Repos',
+    style: 'dashboard.css',
+    logged_in: req.session.logged_in
+  };
+  
+  if (repositories) {
+    renderData.repositories = repositories; 
   }
+  
+  res.render('search', renderData);
 });
 
 // github issues after repo search
 router.get('/issues', withAuth, (req, res) => {
-  res.render('issues', {
+  let issues = [];
```

Comment:
I think this and the next line can be changed to:

let issues = req.session.issues || [];